libjcat/jcat-blob.c

```diff
@@ -115,6 +127,12 @@ jcat_blob_kind_to_filename_ext(JcatBlobKind kind)
 		return "sha256";
 	if (kind == JCAT_BLOB_KIND_SHA1)
 		return "sha1";
+	if (kind == JCAT_BLOB_KIND_MANIFEST)
+		return "manifest";
+	if (kind == JCAT_BLOB_KIND_CHECKPOINT)
+		return "checkpoint";
+	if (kind == JCAT_BLOB_KIND_INCLUSION_PROOF)
+		return "inclusionproof";
```

Comment:
I notice this doesn't have a hyphen like the ones above do, is that intended?

Reply:
yup, it's designed as an "extension" i.e. like `.txt` and I think the convention there is not to include underscores or hyphens.